tgui/packages/tgui/interfaces/DestinationTagger.js

```diff
@@ -0,0 +1,48 @@
+import { useBackend } from '../backend';
+import { Button, Section, Box, Flex } from '../components';
+import { LabeledListItem } from '../components/LabeledList';
+import { Window } from '../layouts';
+
+export const DestinationTagger = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    destinations,
+    selected_destination,
+  } = data;
+
```

Comment:
if storing the `selected_destination_id` in the data, and not the destination itself, you can get the destination via
```js
  const {
    destinations,
    selected_destination_id,
  } = data;
  const selected_destination = destinations[selected_destination_id];
```

Reply:
This may be a better solution hmm